specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/2024-08-22-preview/confidentialledger.json

```
@@ -0,0 +1,1668 @@
+      "properties": {
+        "ledgerUsers": {
+          "items": {
+            "$ref": "#/definitions/LedgerUser"
+          },
+          "type": "array"
+        },
+        "nextLink": {
+          "description": "Path from which to retrieve the next page of results.",
+          "type": "string"
+        }
+      }
+    },
+    "TransactionId": {
+      "description": "A unique identifier for the state of the ledger. If returned as part of a LedgerEntry, it indicates the state from which the entry was read.",
+      "type": "string"
+    },
+    "TransactionReceipt": {
+      "description": "A receipt certifying the transaction at the specified id.",
+      "required": [
+        "state",
+        "transactionId"
+      ],
+      "properties": {
+        "applicationClaims": {
+          "$ref": "#/definitions/ApplicationClaims"
+        },
+        "receipt": {
+          "$ref": "#/definitions/ReceiptContents"
+        },
+        "state": {
+          "$ref": "#/definitions/LedgerQueryState"
+        },
+        "transactionId": {
+          "$ref": "#/definitions/TransactionId"
+        }
+      },
+      "type": "object"
+    },
+    "TransactionState": {
+      "type": "string",
+      "description": "Represents the state of the transaction.",
+      "enum": [
+        "Committed",
+        "Pending"
+      ],
+      "x-ms-enum": {
+        "name": "TransactionState",
+        "modelAsString": true
+      }
+    },
+    "TransactionStatus": {
+      "description": "Response returned to a query for the transaction status",
+      "required": [
+        "state",
+        "transactionId"
+      ],
+      "properties": {
+        "state": {
+          "$ref": "#/definitions/TransactionState"
+        },
+        "transactionId": {
+          "$ref": "#/definitions/TransactionId"
+        }
+      },
+      "type": "object"
+    },
+    "UserId": {
+      "description": "Identifier for the user. This must either be an AAD object id or a certificate fingerprint.",
+      "type": "string"
+    },
+    "Bundle": {
+      "description": "bundle for the user defined endpoints",
+      "properties": {
+        "metadata": {
+          "$ref": "#/definitions/Metadata"
+        },
+        "modules": {
+          "$ref": "#/definitions/ModuleDef_array"
+        }
+      },
+      "required": [
+        "modules",
+        "metadata"
+      ],
+      "type": "object"
+    },
+    "Metadata": {
+      "properties": {
+        "endpoints": {
+          "$ref": "#/definitions/string_to_string_to_EndpointProperties"
+        }
+      },
+      "required": [
+        "endpoints"
+      ],
+      "type": "object"
+    },
+    "string_to_string_to_EndpointProperties": {
+      "path": {
+        "$ref": "#/definitions/string_to_EndpointProperties"
+      },
+      "type": "object"
+    },
+    "string_to_EndpointProperties": {
+      "map_method_properties": {
+        "$ref": "#/definitions/EndpointProperties"
+      },
+      "type": "object"
+    },
```

This schema is invalid and causing the PR tooling to crash. I'm not sure what should be here, but it might be:
```suggestion
    "string_to_EndpointProperties": {
      "properties": {
        "map_method_properties": {
          "$ref": "#/definitions/EndpointProperties"
        },
      },
      "type": "object"
    },
```